cpp/oneapi/dal/algo/linear_kernel/common.cpp

```diff
@@ -20,28 +20,28 @@ namespace oneapi::dal::linear_kernel {
 
 class detail::descriptor_impl : public base {
 public:
-    double k = 1.0;
-    double b = 0.0;
+    double scale = 1.0;
+    double shift = 0.0;
 };
 
 using detail::descriptor_impl;
 
 descriptor_base::descriptor_base() : impl_(new descriptor_impl{}) {}
 
-double descriptor_base::get_k() const {
-    return impl_->k;
+double descriptor_base::get_scale() const {
+    return impl_->scale;
 }
 
-double descriptor_base::get_b() const {
-    return impl_->b;
+double descriptor_base::get_shift() const {
+    return impl_->shift;
 }
 
-void descriptor_base::set_k_impl(const double value) {
-    impl_->k = value;
+void descriptor_base::set_scale_impl(const double value) {
```

Comment:
In my point of view, if `descriptor_base` is in public namespace (`oneapi::dal::linear_kernel`), when user potentially may use it and we have to have a docs for it. Then, name setters `set_something_impl()` is not appropriate in this case, since it's public API. Moreover, it is inconsistent with getter names: (`set_scale_impl()` but `get_scale()` in `descriptor base`). I think we can move descriptor_base to details and name all of its methods consistently, with *_impl* or without it. It's how it done in data management - please see `table`, `homogen_table`, `row_accessor` for example.